packages/blockchain-wallet-v4-frontend/src/data/signup/slice.ts

```diff
@@ -30,7 +35,19 @@ const signupSlice = createSlice({
   initialState,
   name: 'signup',
   reducers: {
+    accountRecoveryVerify: (state, action?) => {},
+    accountRecoveryVerifyFailure: (state, action) => {
```

Comment:
is it possible to pass the type of action.payload? `(state, action: PayloadAction<AcountRecoveryVerifyPayload>)`